fix(i18n): add missing translation calls to some places

TFSThiagoBR98 opened the pull request:
<!--

Some key notes before you open a PR:

 1. Select which branch should this PR be merged in?
 2. PR name follows [convention](http://karma-runner.github.io/4.0/dev/git-commit-msg.html)
 3. All tests pass locally, UI and Unit tests
 4. All business logic and validations must be on the server-side
 5. Update necessary Documentation
 6. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes


Also, if you're new here

- Documentation Guidelines => https://github.com/frappe/erpnext/wiki/Updating-Documentation

- Contribution Guide => https://github.com/frappe/frappe/blob/develop/.github/CONTRIBUTING.md

- Pull Request Checklist => https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist

-->

This fixes some places where there is no translation function

<!-- You can skip this if you're fixing a typo or updating existing documentation -->

This adds translation for: Tour Modals, View Selector, Workspace Headers, List Settings and Kanban Settings

<!-- Example: When "Adding a function to do X", explain why it is necessary to have a way to do X. -->



<!-- Add images/recordings to better visualize the change: expected/current behviour -->

TFSThiagoBR98 committed:
fix(i18n): add missing translations to some places

ankush commented on frappe/public/js/frappe/list/base_list.js:
```diff
@@ -199,11 +199,11 @@ frappe.views.BaseList = class BaseList {
 		if (frappe.boot.desk_settings.view_switcher && !this.meta.force_re_route_to_default_view) {
 			/* @preserve
 			for translation, don't remove
-			__("List View") __("Report View") __("Dashboard View") __("Gantt View"),
```
Read the comment above :grimacing: It's explicitly kept like this for translation extraction. Source strings won't be extractable after your change. 


Also, languages aren't that simple so `translation of view type` + `translation of "view"` != translation of entire string. 


Lets revert this change.

TFSThiagoBR98 replied:
@ankush, Thanks for reviewing. I think I found the problem, when I was doing the translation, I executed the 'bench get-untranslated' command that generated a list of untranslated texts that included `{0} View`, with the translation of this text caused the information in the selector to be partially translated.

As I understand it, the execution of this function looks like this: `__(__('{0} View', [this.view_name]))`, if the text `{0} View` is translated, it breaks the translation afterwards, I think it is better to replace it with `$.format('{0} View', [this.view_name])`

Without `{0} View` Translation:
![image](https://github.com/frappe/frappe/assets/16943898/76dc66c6-1288-4217-9dba-0fbf33775b6a)

With `{0} View` Translation to `Visualização em {0}`:
![image](https://github.com/frappe/frappe/assets/16943898/0d7a14ed-fb4a-4cd4-a9c1-8ceb24987493)

TFSThiagoBR98 committed:
fix(i18n): Replace the translation function with String Format

To avoid run the translation function twice and prevent the string `{0} View` from being extracted and translated

TFSThiagoBR98 committed:
Merge branch 'frappe:develop' into fix-i18n-missing

TFSThiagoBR98 committed:
Merge branch 'frappe:develop' into fix-i18n-missing

TFSThiagoBR98 committed:
Merge branch 'frappe:develop' into fix-i18n-missing

TFSThiagoBR98 committed:
Merge branch 'develop' into fix-i18n-missing

TFSThiagoBR98 committed:
fix: add translation strings to form tour

TFSThiagoBR98 committed:
fix

TFSThiagoBR98 committed:
fix
